web-registry/src/pages/CreditClassDetails.tsx

```diff
@@ -48,21 +48,23 @@ function CreditClassDetail({ isLandSteward }: CreditDetailsProps): JSX.Element {
   });
 
   useEffect(() => {
-    if (creditClassId) {
-      queryEcoClassInfo(creditClassId)
-        .then(res => {
+    const fetch = async (): Promise<void> => {
```

Comment:
i added this so it was a bit more consistent with the fetch that happens in BatchDetails.tsx. but i also wanted to convert this to make use of the async/await style. i saw that we were also using that style in BatchDetails.tsx and i thought it might be good to be consistent.